platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_ssd_upgrade_status.sh

```diff
@@ -93,6 +103,7 @@ else
         touch $SSD_FW_UPGRADE/GPIO7_pending_upgrade
```

Comment:
what if the SSD fw is OLD and still 101 line returns False,  shouldn't we ` touch $SSD_FW_UPGRADE/GPIO7_pending_upgrade` in that case?

Reply:
SSD_UPGRADE_STATUS1 and SSD_UPGRADE_STATUS2 are ff in default.
Hence, it wont get False incase no SSD attempt is attempted